src/perform_step/low_order.jl

```diff
@@ -229,3 +229,124 @@ end
 
   end
 end
+
+@muladd function perform_step!(integrator,cache::RKMil_GeneralConstantCache,f=integrator.f)
+  @unpack t,dt,uprev,u,W,p = integrator
+  dW = W.dW
+
+  if integrator.alg.is_commutative
+    WikJ = 1//2 .* vec(dW) .* vec(dW)'
```

Comment:
I think this could be handled as a branch in the iterated I just like diagonal?

Reply:
But there we have a function for diagonal noise and the commutative noise it's alg specific

Reply:
The way to generate WikJ for commutative noise isn't alg-specific.

Reply:
No not that. I mean we don't have an efficient way to check for commutative noise like we have for diagonal noise so rn commutative noise is just an option in the alg struct

Reply:
I see. Make it a value type so that we can dispatch on it and do a cleaner WikJ construction, and we should find a better system for noise classifications later.